k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-data/templates/cortx-data-pod.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         cortx.io/machine-id: {{ .Values.cortxdata.machineid.value }}
         {{- end }}
         cortx.io/service-type: cortx-data
+        cortx.io/hax-enabled: "true"
```

Comment:
Just curious... is there some significance to `true` being in quotes?  I see that in the line above `cortx-data` does not have quotes around it.  (Maybe quotes are best practice, but you're not changing other lines?)

Reply:
The right-hand side of the Labels are strings. They should always be quoted, but k8s / Go make it easy to recognize when there's a String there and automatically convert it - but it can't do that with `true` or `false` since boolean types are a thing in Go. So you have to quote them when using "true" or "false" in Label values.

I'll go back and add quotes to the labels we have in here. We should make sure we are quoting all of our Labels as best practice as part of @keithpine work in the Labels issue.